cvat-data/package.json

```diff
@@ -23,7 +23,8 @@
     "webpack-dev-server": "^3.8.0"
   },
   "dependencies": {
-    "jszip": "3.1.5"
+    "jszip": "3.1.5",
+    "jpeg-js": "0.3.6"
```

Comment:
Could you please instead of jpeg-js try https://github.com/AntoineViau/webassembly-jpeg? Basically the repository contains instructions how to compile libjpeg to a web assembly module. Let try and compare speed?